server/controller/handler/handlers.go

```diff
@@ -66,33 +56,27 @@ func FileHandler(w http.ResponseWriter, r *http.Request) {
 	}
```

Comment:
here also you have to return some error message in response

Reply:
Can you exemplify  @chandankumar4 ?

Reply:
in the response return the error message like: "file not found"